packages/runtime/container-runtime/src/blobManager.ts

```diff
@@ -524,6 +519,17 @@ export class BlobManager extends TypedEventEmitter<IBlobManagerEvents> {
 		entry.handleP.resolve(this.getBlobHandle(localId));
 	}
 
+	/**
+	 * sendBlobAttachOp with logging for expired blobs
+	 */
+	private sendBlobAttach(localId: string, blobId?: string): void {
```

Comment:
why create a new method? and not just put this in sendBlobAttachOp. having too very similar methods adds the possibility for confusion and future developers calling the wrong method.

Reply:
i see one is a member. i add the member explicitly: `private readonly sendBlobAttachOp` and change the construtor to not set it implicitly: `constructor(...,sendBlobAttachOp)` then in the constructor body set the member: 
```
this.sendBlobAttachOp=(localId: string, blobId?: string)=> {
		const pendingEntry = this.pendingBlobs.get(localId);
		if (pendingEntry) {
			this.logBlobAttachInfo(pendingEntry);
		}
		return sendBlobAttachOp(localId, blobId);
}
```
this will only leave a single thing to call, and prevent miss calls